src/lib/openapi/spec/create-strategy-variant-schema.ts

```diff
@@ -0,0 +1,55 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const createStrategyVariantSchema = {
+    $id: '#/components/schemas/createStrategyVariantSchema',
+    type: 'object',
+    additionalProperties: true,
+    description:
+        '[WIP] A strategy variant allows to attach any data to strategies instead of only returning `true`/`false`. Strategy variants take precedence over feature variants.',
+    required: ['name', 'weight', 'weightType', 'stickiness'],
+    properties: {
+        name: {
+            type: 'string',
+            description: 'The variants name. Is unique for this feature toggle',
```

Comment:
Out of curiosity: do we use the same variants as for strategies or do you now have two different sets of variants (one for strategies and one for the feature)?

```suggestion
            description: 'The variant name. Must be unique for this feature toggle',
```